### PR TITLE
Fixes third-party links title

### LINKED ISSUE #8967: Third party links title incorrect size
**Describe the bug**
The size of the title is slightly incorrect -
1008+px (Russian & Arabic) - title is 24px, design specifies 20px
240-600px (Russian & Arabic) - title is 20px, design specifies 18px

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/russian/podcasts/p05607v8 or https://www.bbc.com/arabic/podcasts/p08z9x26
2. Scroll down to Third Party Links
3. See error

**Expected behaviour**
Title should be to spec

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### LINKED ISSUE #8981: Podcasts 3rd party links - Add english lang attribute to english 3rd party names
**Describe the bug**
Ensure english 3rd party names are announced with the correct accent. 

**Expected behaviour**
English 3rd party names are announced with the correct accent and the programme title is annouced with correct accent e.g. that of the service language.

**Screenshots**
Add lang attribute as per screen shot

Screenshot 2021-03-15 at 16 32 43

### PR DESCRIPTION
**Pull request by mukama**
Resolves https://github.com/bbc/simorgh/issues/8967
Resolves https://github.com/bbc/simorgh/issues/8981

**Overall change:**
Updates the third-party link title size. Also adds `lang` to third-party links.

**Code changes:**
- Replaces `SectionLabel` component with `styled.h2` because `SectionLabel` uses different font specs when compared to the third-party links title [in the design](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/6021542152ff99146b76477d).
- Adds `lang` attribute to third party links 
- Updates corresponding snapshots.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -15,145 +15,61 @@ exports[`PodcastExternalLinks Should render external links 1`] = `
...
-  padding-right: 0.5rem;
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-align-items: center;
-  -webkit-box-align: center;
-  -ms-flex-align: center;
-  align-items: center;
+  color: #222222;
```
Comment (greenc05) [inline src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: We shouldn't be using the link colour as this isn't a link

File: src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -253,19 +152,19 @@ exports[`PodcastExternalLinks Should render external links 1`] = `
           >
             Apple
```
Comment (greenc05) [inline src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: Would it be a good idea to add the lang attrributes as part of this PR too? https://github.com/bbc/simorgh/issues/8981

File: src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (mukama) [reply src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: Thank you for the review @greenc05. It looks from the Zeplin designs that the color is `#222222` - See https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5fbc0b18dbfa4a5b7b118549

File: src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap (hunk 2 above)
Comment (mukama) [reply src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: Absolutely. Let me fix and merge https://github.com/bbc/simorgh/issues/8981 in here as well. Nice catch @greenc05

Comment (mukama) [reply src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: @greenc05 this issue ha been addressed

Comment (greenc05) [reply src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: Looks good thanks @mukama I assume that we specifiy the lang attribute as english somewhere? We should check if all the 3rd parties have english names or if it's just some of them @gavinspence?

File: src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (greenc05) [reply src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: Thanks for updating as per Emily's confirmation

File: src/app/containers/PodcastExternalLinks/index.jsx [hunk 3]
```diff
@@ -79,22 +83,20 @@ const PodcastExternalLinks = ({ brandTitle, links }) => {
...
-      </StyledSectionLabel>
+      </ThirdPartyLinksTitle>
       {hasMultipleLinks ? (
         <StyledList role="list">
           {links.map(({ linkText, linkUrl }) => (
             <StyledListItem dir={dir} key={linkText}>
               <Link href={linkUrl} service={service} script={script} dir={dir}>
                 <span role="text">
-                  {linkText}
+                  <span lang={lang}>{linkText}</span>
```
Comment (rebeccamcginn) [inline src/app/containers/PodcastExternalLinks/index.jsx:101]: What are your thoughts for adding this span here?

File: src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap (hunk 2 above)
Comment (mukama) [reply src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: Yes, the lang attribute will update to English on english pages :) Thanks for the review @greenc05.

File: src/app/containers/PodcastExternalLinks/index.jsx (hunk 3 above)
Comment (mukama) [reply src/app/containers/PodcastExternalLinks/index.jsx:101]: Hi @rebeccamcginn. @greenc05 advised adding lang separately just for the link text as seen in this example 
image

Comment (rebeccamcginn) [reply src/app/containers/PodcastExternalLinks/index.jsx:101]: Cool. We don't need to pass down the language from `OnDemandAudioPage`; we can get it from serviceContext in `PodcastExternalLinks`, line 71

Comment (mukama) [reply src/app/containers/PodcastExternalLinks/index.jsx:101]: Wow! Thanks for the heads up. Let me clean it up right away

Comment (mukama) [reply src/app/containers/PodcastExternalLinks/index.jsx:101]: @rebeccamcginn I've updated the PR. Stopped passing `lang` down from parent components and instead used the `serviceContext`

**Review by rebeccamcginn — APPROVED** [review]

File: src/app/containers/PodcastExternalLinks/index.jsx (hunk 3 above)
Comment (simonsinclair) [reply src/app/containers/PodcastExternalLinks/index.jsx:101]: Hey @mukama, is this treatment also needed on [L116](https://github.com/bbc/simorgh/blob/01db1e971ef0c151556484bf7c66b7de21829e29/src/app/containers/PodcastExternalLinks/index.jsx#L116)?

Comment (mukama) [reply src/app/containers/PodcastExternalLinks/index.jsx:101]: I believe so. Nice catch @simonsinclair

Comment (mukama) [reply src/app/containers/PodcastExternalLinks/index.jsx:101]: I've made the update @simonsinclair

**Review by simonsinclair — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
LGTM.. 
* Checked that english lang attribute is added to to english links
* Third party links are displayed in correct size as per spec

**Pull request merged**